0000-template.md

```diff
@@ -0,0 +1,47 @@
+- Feature Name: (fill with a unique ident, my_awesome_feature)
+- Start Date: (fill with today's date, DD-MM-YYYY)
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Brief (one-paragraph) explanation of the feature.
+
+# Motivation
+[motivation]: #motivation
+
+- Why are we doing this?
+- What use cases does it support?
+- What is the expected outcome?
+
+Describe the problem you are trying to solve, and its constraints, without coupling them too closely to the solution you have in mind.
+> If this RFC is not accepted, the motivation can be used to develop alternative solutions.
```

Comment:
```suggestion
If this RFC is not accepted, the motivation can be used to develop alternative solutions.
```

Reply:
Why this change - just a matter of styling?

Reply:
Sort of - in Markdown `>` means quoting, which is not the expected semantics here I guess.

Reply:
That's right, the syntax describes it as quoting, but I find it useful, in terms of styling, to refer to tips / additional details / links to learn more.

Reply:
Will commit suggestions